rapidsai/devel-centos.arm64.Dockerfile

```
@@ -82,6 +82,12 @@ RUN gpuci_conda_retry install -y \
       jq \
       mamba \
       rapids-scout-local
+# Install sccache
+RUN wget https://github.com/mozilla/sccache/releases/download/v0.2.15/sccache-v0.2.15-aarch64-unknown-linux-musl.tar.gz \
+    && tar -xzf sccache-v0.2.15-aarch64-unknown-linux-musl.tar.gz \
+    && rm -r sccache-v0.2.15-aarch64-unknown-linux-musl.tar.gz \
+    && cp sccache-v0.2.15-aarch64-unknown-linux-musl/sccache /usr/local/bin \
```

```suggestion
    && tar -xzf sccache-v0.2.15-aarch64-unknown-linux-musl.tar.gz \
    && mv sccache-v0.2.15-aarch64-unknown-linux-musl/sccache /usr/local/bin \
    && rm -rf sccache-v0.2.15-aarch64-unknown-linux-musl.tar.gz sccache-v0.2.15-aarch64-unknown-linux-musl/ \
```

There are other files in the tarball besides just the `sccache` binary. So we'd want to delete the extracted folder when we're done with it. Also, we should switch the `cp` to `mv` so that there are not duplicate `sccache` binaries in the image. I didn't test the above commands, but I think they should work.

---

Ack.